dali/operators/reader/nvdecoder/cuvideodecoder.cc

```diff
@@ -133,13 +133,18 @@ void CUVideoDecoder::reconfigure(unsigned int height, unsigned int width) {
     DALI_ENFORCE(decoder_, "Trying to reconfigure uninitialized decoder");
 
     DALI_ENFORCE(width >= caps_.nMinWidth && height >= caps_.nMinHeight,
-                 "Video is too small in at least one dimension.");
+                 make_String("Video is too small in at least one dimension. Provides size is ",
+                 width, "x", height, " while the decoder supports at least ", caps_.nMinWidth, "x",
+                 caps_.nMinHeight);
 
     DALI_ENFORCE(width <= caps_.nMaxWidth && height <= caps_.nMaxHeight,
-                 "Video is too large in at least one dimension.");
+                 "Video is too large in at least one dimension. Provides size is ",
+                 width, "x", height, " while the decoder supports at most ", caps_.nMaxWidth, "x",
+                 caps_.nMaxHeight);
 
     DALI_ENFORCE(width * height / 256 <= caps_.nMaxMBCount,
-                 "Video is too large (too many macroblocks).");
+                 make_String("Video is too large (too many macroblocks). Provided video has ",
+                 width * height / 256, " while the decoder supports ", caps_.nMaxMBCount);
```

Comment:
```suggestion
                 width * height / 256, " while the decoder supports up to ", caps_.nMaxMBCount, "macroblocks.");
```

Maybe we can also provide the info that this is size related?

Reply:
Done